configs/rec/master/master_resnet31.yaml

```diff
@@ -55,8 +55,8 @@ loss:
 
 scheduler:
   scheduler: warmup_cosine_decay
-  min_lr: 0.00001
-  lr: 0.001
+  min_lr: 0.000005
+  lr: 0.0005
```

Comment:
replace these values with `5.0e-6` and `5.0e-4` for easier reading.

Reply:
yaml does not support scientist format, it will be parsed as string

Reply:
It does, you can refer to [here](https://github.com/mindspore-lab/mindocr/blob/main/configs/det/dbnet/db_r50_icdar15.yaml#L56). But the syntax is very strict, for example, `5e-6` will be parsed as string.

Reply:
updated